### DISCUSSION TITLE
charge voltage restricting

### DISCUSSION THREAD

**Discussion by andrewcrosse** [Louisvdw/dbus-serialbattery#643] [General]
hi all

new to this, but seem to have charge voltage restricting chargers
in vrm charge voltage limit is a steady 53.6v(53.6/16=3.35)
the only place i can find this value is the float value, i altered it too 3.4 and this raised the charge voltage
current soc 87%
cells 3.3-3.4
temp 
degrees

# Charge current control management referring to cell-voltage enable (True/Fals>
CCCM_CV_ENABLE = True
# Discharge current control management referring to cell-voltage enable (True/F>
DCCM_CV_ENABLE = True

# Set Steps to reduce battery current. The current will be changed linear betwe>
CELL_VOLTAGES_WHILE_CHARGING         = [3.55, 3.50, 3.45,3.4, 3.0]
MAX_CHARGE_CURRENT_CV                = [   0,    2,  30, 170, 170]

CELL_VOLTAGES_WHILE_DISCHARGING      = [2.70, 2.80, 2.90, 3.10, 3.65]
MAX_DISCHARGE_CURRENT_CV             = [   0,    5,  30, 210, 210]

CCCM_T_ENABLE = True
# Charge current control management referring to temperature enable (True/False>
DCCM_T_ENABLE = True

TEMPERATURE_LIMITS_WHILE_CHARGING    = [55, 40,  32, 10,  5,  2, 0]
MAX_CHARGE_CURRENT_T                 = [ 0, 28, 170, 170 , 100, 28, 0]

TEMPERATURE_LIMITS_WHILE_DISCHARGING = [55, 40,  35,   5,  0, -20]
MAX_DISCHARGE_CURRENT_T              = [ 0, 28, 210, 210, 28,   0]

MIN_CELL_VOLTAGE = 2.9
MAX_CELL_VOLTAGE = 3.45
FLOAT_CELL_VOLTAGE = 3.4
MAX_VOLTAGE_TIME_SEC = 15*60
SOC_LEVEL_TO_RESET_VOLTAGE_LIMIT = 90

# battery Current limits
MAX_BATTERY_CHARGE_CURRENT = 170.0
MAX_BATTERY_DISCHARGE_CURRENT = 210.0

# Charge current control management enable (True/False).
CCCM_SOC_ENABLE = True
# Discharge current control management enable (True/False).
DCCM_SOC_ENABLE = True

#charge current soc limits
CC_SOC_LIMIT1 = 98
CC_SOC_LIMIT2 = 95
CC_SOC_LIMIT3 = 91

#charge current limits
CC_CURRENT_LIMIT1 = 5
CC_CURRENT_LIMIT2 = MAX_BATTERY_CHARGE_CURRENT/4
CC_CURRENT_LIMIT3 = MAX_BATTERY_CHARGE_CURRENT/2

#discharge current soc limits
DC_SOC_LIMIT1 = 10
DC_SOC_LIMIT2 = 20
DC_SOC_LIMIT3 = 30

#discharge current limits
DC_CURRENT_LIMIT1 = 5
DC_CURRENT_LIMIT2 = MAX_BATTERY_DISCHARGE_CURRENT/4
DC_CURRENT_LIMIT3 = MAX_BATTERY_DISCHARGE_CURRENT/2

# Charge voltage control management enable (True/False).
CVCM_ENABLE = True

# Simulate Midpoint graph (True/False).
MIDPOINT_ENABLE = False

#soc low levels
SOC_LOW_WARNING = 20
SOC_LOW_ALARM = 10

**Comment by mr-manuel** [comment]
Hello, why did you not open a new support request under issues? Please do that.
